config/sync/extlink.settings.yml

```diff
@@ -20,7 +20,7 @@ extlink_use_font_awesome: false
 extlink_font_awesome_classes:
   links: 'fa fa-external-link'
   mailto: 'fa fa-envelope-o'
-extlink_icon_placement: append
+extlink_icon_placement: prepend
```

Comment:
This impacts screen readers. I believe it would be more effective to keep it as `append` and manipulate the styles for the mailto to position the icon before.